hasher-matcher-actioner/docker-compose.yaml

```diff
@@ -1,40 +1,55 @@
-version: "3"
-
 x-postgres-variables: &postgres-variables
-  POSTGRES_USER: media_match
+  POSTGRES_USER: postgres
   # TODO: Use a secret instead.
-  POSTGRES_PASSWORD: hunter2
-  POSTGRESS_HOST: db
-  POSTGRESS_DBNAME: media_match
+  POSTGRES_PASSWORD: postgres
+  POSTGRES_HOST: postgresql
+  POSTGRES_DB: media_match
 
 services:
-  app:
-    build: .
+  postgresql:
+    image: postgres:17-alpine
     restart: unless-stopped
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
     ports:
-      - 5000:5000
+      - 5432:5432
+    environment:
+      <<: *postgres-variables
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U $$POSTGRES_USER -d $$POSTGRES_DB"]
+      interval: 1s
+      retries: 5
+      start_period: 5s
+      timeout: 10s
+    networks:
+      - backend
+
+  migrations:
+    build: .
+    restart: no
     environment:
       <<: *postgres-variables
-      OMM_CONFIG: /build/reference_omm_configs/development_omm_config.py
+      OMM_CONFIG: /reference_configs/development_omm_config.py
```

Comment:
```suggestion
      OMM_CONFIG: /reference_omm_configs/development_omm_config.py
```

Reply:
This is actually already correct, see the copy in the dockerfile